libs/foundry-dev-tools/src/foundry_dev_tools/clients/multipass.py

```diff
@@ -13,25 +14,54 @@
     from collections.abc import Iterator
 
     import requests
+
 DEFAULT_TOKEN_LIFETIME_IN_SECONDS = 604800
-"""The default token lifetime in seconds which is equivalent to seven days."""
+"""The default token lifetime in seconds which is equivalent to seven days (7*24*60*60)."""
 
 MINIMUM_TOKEN_PAGE_SIZE = 0
 DEFAULT_TOKEN_PAGE_SIZE = 100
 MAXIMUM_TOKEN_PAGE_SIZE = 500
 
+MINIMUM_MAX_DURATION_IN_SECONDS = 0
+"""The minimum max duration for expiration settings which is equivalent to zero seconds."""
+DEFAULT_MAX_DURATION_IN_SECONDS = 604800
+"""The default max duration for expiration settings which is equivalent to seven days (7*24*60*60)."""
+
+
+def _validate_timezone(dt: datetime, variable_name: str) -> datetime:
+    """Checks whether a datetime object contains timezone information and if not so, assigns 'UTC' timezone.
+
+    Args:
+        dt: The datetime for which to check the presence of time zone information
+        variable_name: The name of the variable to inform the user in case of a malformed datetime object
+
+    Returns:
+        datetime:
+            the datetime object with the time zone information
+    """
+    if not dt.tzinfo:
+        msg = (
+            f"{dt.isoformat(timespec='seconds')} passed for parameter '{variable_name} is missing timezone information."
+            f"Ensure timezone is provided for datetime object. Defaults to '{timezone.utc!s}' timezone!"
+        )
```

Comment:
```suggestion
        msg = (
            f"{dt.isoformat(timespec='seconds')} passed for parameter '{variable_name} is missing timezone information."
            f"Setting the timezone to '{timezone.utc!s}'."
            f"Ensure timezone is provided for datetime object."
        )
```

Reworded it to hopefully make it more clear that the code automatically sets the timezone to UTC